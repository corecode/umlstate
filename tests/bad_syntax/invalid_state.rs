use umlstate::umlstate;

umlstate! {
    machine Foo {
        state a;
        A + E => B;
    }
}

fn main() {}
