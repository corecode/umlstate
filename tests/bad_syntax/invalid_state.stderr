error: transition target is not a declared state
 --> tests/bad_syntax/invalid_state.rs:6:18
  |
6 |         A + E => B;
  |                  ^
